stream_chat/async_chat/client.py

```diff
@@ -332,7 +332,11 @@ async def update_message_partial(
     async def delete_message(self, message_id: str, **options: Any) -> StreamResponse:
         return await self.delete(f"messages/{message_id}", options)
 
-    async def get_message(self, message_id: str) -> StreamResponse:
+    async def get_message(
+        self, message_id: str, show_deleted_message: bool = False
```

Comment:
Same as the js sdk, better to add a options object:
```suggestion
        self, message_id: str, **options: Any
```

Something along those lines.